databricks/koalas/window.py

```diff
@@ -30,8 +30,26 @@ class _RollingAndExpanding(object):
 
 
 class Rolling(_RollingAndExpanding):
-    def __init__(self, obj):
-        self.obj = obj
+    def __init__(self, kdf_or_kser, window, min_periods=None):
+        from databricks.koalas import DataFrame, Series
+        from databricks.koalas.groupby import SeriesGroupBy, DataFrameGroupBy
+        window = window - 1
+        min_periods = min_periods if min_periods is not None else 0
+
+        if window < 0:
+            raise ValueError("window must be >= 0")
+        if (min_periods is not None) and (min_periods < 0):
+            raise ValueError("min_periods must be >= 0")
+        self._window_val = window
+        self._min_periods = min_periods
+        self.kdf_or_kser = kdf_or_kser
+        if not isinstance(kdf_or_kser, (DataFrame, Series, DataFrameGroupBy, SeriesGroupBy)):
+            raise TypeError(
+                "kdf_or_kser must be a series or dataframe; however, got: %s" % type(kdf_or_kser))
+        if isinstance(kdf_or_kser, (DataFrame, Series)):
+            self._index_scols = kdf_or_kser._internal.index_scols
+            self._window = Window.orderBy(self._index_scols).rowsBetween(
+                Window.currentRow - window, Window.currentRow)
```

Comment:
You can simply send `-window` instead of `Window.currentRow - window` IIRC.

Reply:
thanks for the comment! what a convenient.